src/net.cpp

```diff
@@ -1960,7 +1960,7 @@ void CConnman::ThreadOpenAddedConnections()
 }
 
 // if successful, this moves the passed grant to the constructed node
-void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFailure, CSemaphoreGrant *grantOutbound, const char *pszDest, bool fOneShot, bool fFeeler, bool manual_connection)
+void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFailure, CSemaphoreGrant *grantOutbound, const char *pszDest, bool fOneShot, bool fFeeler, bool fManualConnection)
```

Comment:
FYI the style guide prefers the prior style (despite the inconsistency here):
"Variable and namespace names are all lowercase, and may use `_` to separate words (snake_case)"